### PR TITLE
Scripts for computing amplitude and phase maps

### PR DESCRIPTION
**Pull request by akhoubani**

### CODE REVIEW

**Comment by paugier** [comment]
Thanks! 

- [ ] Could you please put these files is a directory in `doc/scripts` with a meaningful name?
- [ ] Could you add a short README.md file explaining how to use these scripts and with which scripts the simulations have been launched?

You only use these 2 files to produce all the data for the article ?

File: doc/scripts/2022sidewall_conv_instabilities/find_jet.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+def find_jet_legths(field_base, prandtl, aspect_ratio):
```
Comment (paugier) [inline doc/scripts/2022sidewall_conv_instabilities/find_jet.py]: jet_lengths

**Comment by paugier** [comment]
Could you please move the files to produce figures and supplementary materials in a subdirectory?

**Comment by paugier** [comment]
Thanks. Do you consider it ready? Can I merge?

**Comment by akhoubani** [comment]
> Thanks. Do you consider it ready? Can I merge?

I guess it is ready to be merged.

**Comment by paugier** [comment]
Can you please add the url https://github.com/snek5000/snek5000-cbox/tree/main/doc/scripts/2022sidewall_conv_instabilities in the description of the Zenodo dataset?

**Comment by paugier** [comment]
And by the way, you can also add the Arxiv link... https://arxiv.org/abs/2304.12657

**Pull request merged**